tests/image_test.py

```diff
@@ -1359,8 +1359,11 @@ def test_validate_filename(self, tmpdir):
         assert not image.validate_filename('some_test_file.png', valid_extensions)
 
     def test_iterator_empty_directory(self):
-        iterator = image.Iterator(0, 0, False, 0)
-        next(iterator.index_generator)
+        # Testing with randomly chosen batch sizes
```

Comment:
The batches are not generated randomly, something like: "Testing with different batch sizes" is a best description. Also there is no need to check with more batch sizes, only 0 and non-zero.

Reply:
Done.